pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nextpy"
-version = "0.3.3"
+version = "0.3.5"
```

Comment:
Why the version bump?

Reply:
Jupiter has been released in version```0.3.4``` Next, we're releasing the state-management update in version ```0.3.5```